nrpy/infrastructures/BHaH/CurviBoundaryConditions/CurviBoundaryConditions.py

```diff
@@ -1299,17 +1321,21 @@ def register_CFunction_apply_bcs_outerradiation_and_inner(
   //              STEP 2 OF 2.
   apply_bcs_inner_only(commondata, params, bcstruct, rhs_gfs); // <- apply inner BCs to RHS gfs only
 """
-    cfc.register_CFunction(
```

Comment:
This should never happen; the list of coordinate systems passed to list_of_CoordSystems must be unique, and checking for uniqueness in `CurviBoundaryConditions_register_C_functions()` is the correct action.